modules/katamari/src/test/ts/atomic/api/fun/FunTest.ts

```diff
@@ -5,46 +5,61 @@ import fc from 'fast-check';
 import * as Fun from 'ephox/katamari/api/Fun';
 
 describe('atomic.katamari.api.fun.FunTest', () => {
-  it('unit tests', () => {
+  it('TINY-7494: should compose functions correctly', () => {
```

Comment:
I don't think `TINY-7494` is the correct reference here, yes it's the ticket you're doing but all that says is "split the test up". Even 7469, which was the precursor, isn't correct because that was "convert tests from the old framework to this new one".

JIRA references are used to explain why the test exists; I'd rather have no reference than a misleading one.